README.md

```diff
@@ -1,12 +1,26 @@
-# Blood Donor - Django Backend
+# Blood Donor - Server
 
 The main REST API backend responsible for serving stateful data for the Blood Donor mobile apps.
 
-# Building & Running
-
-If you're building this project for the first time, you'll need [Docker](https://docs.docker.com/get-docker/).
+### Building & Running
```

Comment:
According to [markdownlint - MD001 - Heading levels should only increment by one level at a time](https://github.com/DavidAnson/markdownlint/blob/main/doc/md001.md), you should decrease the heading level to `##`.